README.md

```diff
@@ -1,4 +1,4 @@
-# TODO Library 
```

Comment:
Could add a small unrelated fix to the MR? I thought it shouldn't affect linter and tests and we would be able to do it later. But it seems I was wrong. I'm talking about this fix(replace deprecated `chrono::NativeDateTime` method call  with a newer one):

```
error: use of deprecated associated function `chrono::NaiveDateTime::from_timestamp_opt`: use `DateTime::from_timestamp` instead
Error:   --> src/timer.rs:49:51
   |
49 |             let dt_start = chrono::NaiveDateTime::from_timestamp_opt(n, 0)?;
   |                                                   ^^^^^^^^^^^^^^^^^^
   |
   = note: `-D deprecated` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(deprecated)]`
```

Thank you!

Reply:
Yes, I've seen this warning and I though it just needs a quick fix, but `chrono` wanted us to migrate from naive dates to UTC, and I was not sure how to properly test it so I left it out at first. 

[I've just updated the code](https://github.com/alexesmet/todo_lib/commit/60e18214d627e661e602bec178121b9a80975113), it does not look like something that could break, but I also can't be 100% sure :sweat_smile: 

Please take a look and resolve this thread if it's ok